package.json

```diff
@@ -11,7 +11,7 @@
     "./": "./"
   },
   "homepage": "https://www.opensearch.org/",
-  "version": "1.0.2",
+  "version": "1.1.0",
```

Comment:
is this planned ?

Reply:
I am not aware of the planning yet. following up on @kavilla 's comment.

Reply:
I am not aware of the planning yet. following up on @kavilla 's comment.